layers/meta-balena-raspberrypi/conf/machine/revpi-connect-s.conf

```diff
@@ -0,0 +1,12 @@
+#@TYPE: Machine
+#@NAME: Revolution Pi Connect
+#@DESCRIPTION: Machine configuration for Revolution Pi Connect board
+
+require conf/machine/include/revpi4.inc
+
+KERNEL_DEVICETREE:append = " \
+	overlays/revpi-connect.dtbo \
+	overlays/revpi-connect-dt-blob-overlay.dtb \
+	overlays/revpi-connect-can.dtbo \
```

Comment:
The device trees for `Revolution Pi Connect S` are named exactly the same as the device trees for the older `Revolution Pi Connect` board?

Reply:
Yes it is. should I change this?

Reply:
If it is using the exact same overlay then it's fine. I thought there may be a specific overlay for the S version.